### DISCUSSION TITLE
add animated introduction to v23

### DISCUSSION THREAD

**Discussion by Sonic1paRadise** [linuxdeepin/developer-center#3444] [Features Request & Ideas | 特性请求 & 头脑风暴]
similar to vid I sent

https://user-images.githubusercontent.com/51707249/192464998-39a1576b-1d35-4d4e-9c32-639d6e8fdda9.mp4

for OOBE, not desktop setup!!!
as well as animated icons similar to win11 oobe.
and option to add more than 1 keyboard layout and test it(still having ptsd after keyboard did not work correctly in 20.6's oobe).
as well as desktop setup will add desktop parts step by step
(hover over an option to see the preview and select it to appear actually)
p.s. the desktop setup idea is similar to
"Everything you say will become the truth" sentence

**Comment by BLumia** [comment]
> still having ptsd after keyboard did not work correctly in 20.6's oobe

Does it works with 20.7 “oobe” stage? If not, consider filing a bug so we can look into it. Thanks!

**Comment by Sonic1paRadise** [comment]
well it worked fine in 20.7 oobe

вт, 27 вер. 2022 р. о 12:32 Gary Wang ***@***.***> пише:
